Build failure: brlaser

### Nixpkgs version

- Unstable (25.11)

### Steps to reproduce

Build package on the latest nixpkgs conmmit.
```
github:nixos/nixpkgs/8913c168d1c56dc49a7718685968f38752171c3b?narHash=sha256-TXnlsVb5Z8HXZ6mZoeOAIwxmvGHp1g4Dw89eLvIwKVI%3D (2025-10-06 06:46:10)
```

### Can Hydra reproduce this build failure?

Yes, Hydra can reproduce this build failure.

### Link to Hydra build job

https://hydra.nixos.org/build/308960492

### Relevant log output

```console
Running phase: unpackPhase
unpacking source archive /nix/store/xy7s5pvkf8zrdjcxvnfhh5yn6sw0bdml-source
source root is source
Running phase: patchPhase
Running phase: updateAutotoolsGnuConfigScriptsPhase
Running phase: configurePhase
cmake flags: -DCMAKE_FIND_USE_SYSTEM_PACKAGE_REGISTRY=OFF -DCMAKE_FIND_USE_PACKAGE_REGISTRY=OFF -DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON -DCMAKE_BUILD_TYPE=Release -DBUILD_TESTING=OFF -DCMAKE_INSTALL_LOCALEDIR=/nix/store/xw77ck0psjddffhli4y05ry5xma9hfkz-brlaser-6.2.7/share/locale -DCMAKE_INSTALL_LIBEXECDIR=/nix/store/xw77ck0psjddffhli4y05ry5xma9hfkz-brlaser-6.2.7/libexec -DCMAKE_INSTALL_LIBDIR=/nix/store/xw77ck0psjddffhli4y05ry5xma9hfkz-brlaser-6.2.7/lib -DCMAKE_INSTALL_DOCDIR=/nix/store/xw77ck0psjddffhli4y05ry5xma9hfkz-brlaser-6.2.7/share/doc/brlaser -DCMAKE_INSTALL_INFODIR=/nix/store/xw77ck0psjddffhli4y05ry5xma9hfkz-brlaser-6.2.7/share/info -DCMAKE_INSTALL_MANDIR=/nix/store/xw77ck0psjddffhli4y05ry5xma9hfkz-brlaser-6.2.7/share/man -DCMAKE_INSTALL_INCLUDEDIR=/nix/store/xw77ck0psjddffhli4y05ry5xma9hfkz-brlaser-6.2.7/include -DCMAKE_INSTALL_SBINDIR=/nix/store/xw77ck0psjddffhli4y05ry5xma9hfkz-brlaser-6.2.7/sbin -DCMAKE_INSTALL_BINDIR=/nix/store/xw77ck0psjddffhli4y05ry5xma9hfkz-brlaser-6.2.7/bin -DCMAKE_INSTALL_NAME_DIR=/nix/store/xw77ck0psjddffhli4y05ry5xma9hfkz-brlaser-6.2.7/lib -DCMAKE_POLICY_DEFAULT_CMP0025=NEW -DCMAKE_FIND_FRAMEWORK=LAST -DCMAKE_STRIP=/nix/store/dmypp1h4ldn0vfk3fi6yfyf5yxp9yz0k-gcc-wrapper-14.3.0/bin/strip -DCMAKE_RANLIB=/nix/store/dmypp1h4ldn0vfk3fi6yfyf5yxp9yz0k-gcc-wrapper-14.3.0/bin/ranlib -DCMAKE_AR=/nix/store/dmypp1h4ldn0vfk3fi6yfyf5yxp9yz0k-gcc-wrapper-14.3.0/bin/ar -DCMAKE_C_COMPILER=gcc -DCMAKE_CXX_COMPILER=g++ -DCMAKE_INSTALL_PREFIX=/nix/store/xw77ck0psjddffhli4y05ry5xma9hfkz-brlaser-6.2.7 -DCUPS_SERVER_BIN=lib/cups -DCUPS_DATA_DIR=share/cups
CMake Error at CMakeLists.txt:1 (cmake_minimum_required):
  Compatibility with CMake < 3.5 has been removed from CMake.

  Update the VERSION argument <min> value.  Or, use the <min>...<max> syntax
  to tell CMake that the project requires at least <min> but has been updated
  to work with policies introduced by <max> or earlier.

  Or, add -DCMAKE_POLICY_VERSION_MINIMUM=3.5 to try configuring anyway.


-- Configuring incomplete, errors occurred!
```

### Additional context

_No response_

### System metadata

 - system: `"x86_64-linux"`
 - host os: `Linux 6.12.49, NixOS, 25.11 (Xantusia), 25.11.20251002.7df7ff7`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.31.2`
 - nixpkgs: `/nix/store/9v6qa656sq3xc58vkxslqy646p0ajj61-source`

### Notify maintainers

@onny 

---

**Note for maintainers:** Please tag this issue in your pull request description. (i.e. `Resolves #ISSUE`.)


### I assert that this issue is relevant for Nixpkgs

- [x] I assert that this is a bug and not a support request.
- [x] I assert that this is not a [duplicate of an existing issue](https://github.com/NixOS/nixpkgs/issues?q=is%3Aopen+is%3Aissue+label%3A%220.kind%3A+build+failure%22). 
- [x] I assert that I have read the [NixOS Code of Conduct](https://github.com/NixOS/.github/blob/master/CODE_OF_CONDUCT.md) and agree to abide by it.

### Is this issue important to you?

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

--- a/pkgs/by-name/br/brlaser/package.nix
+++ b/pkgs/by-name/br/brlaser/package.nix
@@ -29,6 +29,8 @@ stdenv.mkDerivation rec {
   ];
 
   cmakeFlags = [
+    # NOTE: this can be removed once version 6.2.8 is released
+    "-DCMAKE_POLICY_VERSION_MINIMUM=3.5"
     "-DCUPS_SERVER_BIN=lib/cups"
     "-DCUPS_DATA_DIR=share/cups"
   ];

